File: frontend/src/app/dto/product/product.update.req.dto.ts
export class ProductUpdateReqDto {
    private id! : number
    private productCode! : string
    private productName! : string
    private ver! : number
    private isActive! : boolean

    public get _productCode(): string {
        return this.productCode;
    }

    public set _productCode(_productCode: string) {
        this.productCode = _productCode;
    }
    
    public get _productName(): string {
        return this.productName;
    }

    public set _productName(_productName: string) {
        this.productName = _productName;
    }
    
    public get _isActive(): boolean {
        return this.isActive;
    }

    public set _isActive(_isActive: boolean) {
        this.isActive = _isActive;
    }

    public get _id(): number {
        return this.id;
    }

    public set _id(_id: number) {
        this.id = _id;
    }

    public get _ver(): number {
        return this.ver;
    }

    public set _ver(_ver: number) {
        this.ver = _ver;
    }
    
}

File: backend/src/main/java/com/lawencon/ticketms/dto/product/ProductUpdateReqDto.java
package com.lawencon.ticketms.dto.product;

public class ProductUpdateReqDto {
	private Long id;
	private String productCode;
	private String productName;
	private Boolean isActive;
	private Integer ver;
	
	public String getProductCode() {
		return productCode;
	}
	public void setProductCode(String productCode) {
		this.productCode = productCode;
	}
	public Boolean getIsActive() {
		return isActive;
	}
	public void setIsActive(Boolean isActive) {
		this.isActive = isActive;
	}
	public Long getId() {
		return id;
	}
	public void setId(Long id) {
		this.id = id;
	}
	public String getProductName() {
		return productName;
	}
	public void setProductName(String productName) {
		this.productName = productName;
	}
	public Integer getVer() {
		return ver;
	}
	public void setVer(Integer ver) {
		this.ver = ver;
	}
}
